<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataDialog</class>
 <widget class="QDialog" name="MetadataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="personnalInformationGroupBox">
     <property name="title">
      <string>Personnal Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="ageGroupLabel">
          <property name="text">
           <string>Age group</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="ageGroupComboBox">
          <item>
           <property name="text">
            <string>N/A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12-17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18-25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>26-40</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>41-60</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&gt;60</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="sexLabel">
          <property name="text">
           <string>Sex</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="sexComboBox">
          <item>
           <property name="text">
            <string>N/A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>M</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>F</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dominantHandLabel">
          <property name="text">
           <string>Dominant Hand</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="dominantHandComboBox">
          <item>
           <property name="text">
            <string>N/A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ambidextrous</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="usedToInteractionSystemsLabel">
          <property name="text">
           <string>Used to video games and interaction systems</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="usedToVideoGamesAndInteractionSystemsCheckBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="notesGroupBox">
     <property name="title">
      <string>Notes</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="notesTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MetadataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MetadataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
